docs/linguistic-data-intro.md

```diff
@@ -5,6 +5,31 @@ the morphodict code to use.
 
 _**Note**: `sssttt` below refers to a [language pair abbreviation](sssttt)._
 
+The *minimum* linguistic data you’ll need to get started on a dictionary
+is the following:
+
+  - The three-letter [ISO 639-3 language
+    codes](https://iso639-3.sil.org/code_tables/639/data) for the source
+    and target languages.
+
+  - The definition of a single word.
+
+  - A paradigm table for that word. This could be as simple as a photocopy
+    of a page from a textbook showing each wordform, annotated with the
+    desired FST analysis for each.
+
+  - It’s nice to know the endonym, i.e., the name of the language in the
```

Comment:
I've noticed some people know _autonym_ but not _endonym_. They mean essentially the same thing.

```suggestion
  - It’s nice to know the endonym, i.e., the autonym or name of the language in the
```

Reply:
Thanks!